terraform/deployment/modules/01-upload-static-files/01-variables.tf

```diff
@@ -0,0 +1,18 @@
+variable "bucket_id" {
+  type = string
+}
+
+variable "build_directory" {
+  type    = string
+  default = "../../frontend/build"
+}
+
+variable "region" {
+  type    = string
+  default = "eu-central-2"
+}
+
+variable "bucket_path_prefix" {
```

Comment:
the others are self-documenting, but you might want to provide a description for this one.